float32 speed
string fan_name
uint32 incubator_id
